### ISSUE TITLE
No border around selects in error state

### ISSUE THREAD

**Issue by akainic** [18F/web-design-standards#1103]
#### Description
The error styling for `input` and `select` is not consistent. In `elements/_inputs.scss`, a red border is applied to `input` with the class `usa-input-error`, but not to `select`. However, `select` does get success styling with the class `usa-input-success`. I am not sure if there is a reason for this styling choice, but the inconsistency looks a bit strange on our form.

#### Ask
It would be great if `select` was added to the `usa-input-error` styles with `input` so the same red border could be applied.

#### Additional Info
<img width="535" alt="screenshot 2016-04-08 11 26 23" src="https://cloud.githubusercontent.com/assets/3886085/14390777/be3212a0-fd86-11e5-9e12-66f93c743b37.png">
<img width="541" alt="screenshot 2016-04-08 11 26 28" src="https://cloud.githubusercontent.com/assets/3886085/14390779/c02d1dc0-fd86-11e5-8868-1c0b0a8ed446.png">

**Comment by amyfens** [comment]
Ditto for `textarea`. We just added another selector to the CSS so that the styling would be applied identically to the general `input` field:

```
.usa-input-error {
  border-left: 4px solid #cd2026;
  margin-top: 3rem;
  padding-bottom: .8rem;
  padding-left: 1.5rem;
  padding-top: .8rem;
  position: relative;
  right: 1.9rem; }
  .usa-input-error input,
  .usa-input-error textarea {
    border: 3px solid #cd2026;
    width: calc(100% + 1.5rem); }
  .usa-input-error label {
    margin-top: 0; }
```

**Comment by maya** [comment]
Thank you @akainic and @amyfens! Yes, I think adding in those error styles for those form fields is a great idea.

**Comment by maya** [comment]
Closed via #1132.
